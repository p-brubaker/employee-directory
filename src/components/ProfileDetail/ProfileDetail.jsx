export default function ProfileDetail({ profile }) {
  return (
    <div>
      <p>{`name: ${profile.name}`}</p>
      <p>{`email: ${profile.email}`}</p>
      <p>{`birthday: ${profile.birthday}`}</p>
      <p>{`bio: ${profile.bio}`}</p>
    </div>
  );
}
