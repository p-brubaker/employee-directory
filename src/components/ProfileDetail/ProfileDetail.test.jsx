import { screen, render } from '@testing-library/react';
import ProfileDetail from '../../components/ProfileDetail/ProfileDetail';

it('should render the profile detail page', async () => {
  const { container } = render(
    <ProfileDetail
      profile={{
        name: 'Paul',
        email: 'pbrubak@gmail.com',
        birthday: '1111/11/11',
        bio: 'Hi!',
      }}
    />
  );

  screen.getByText('name: Paul');
  expect(container).toMatchSnapshot();
});
