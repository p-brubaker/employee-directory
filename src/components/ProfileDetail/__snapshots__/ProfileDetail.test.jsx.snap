// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the profile detail page 1`] = `
<div>
  <div>
    <p>
      name: Paul
    </p>
    <p>
      email: pbrubak@gmail.com
    </p>
    <p>
      birthday: 1111/11/11
    </p>
    <p>
      bio: Hi!
    </p>
  </div>
</div>
`;
